pdk_regression.yml

```diff
@@ -0,0 +1,47 @@
+name: pdk_regression
+channels:
+  - litex-hub
+  - defaults
+dependencies:
+  - _libgcc_mutex=0.1=main
```

Comment:
`klayout` amd `magic` should be enough (unless you really want to pin everything)